dsl/docker-multiarch/views.groovy

```diff
@@ -10,7 +10,9 @@ def images = [
 	'gcc',
 	'haproxy',
 	'hello',
+	'httpd',
 	'irssi',
+	'java',
```

Comment:
Since this specific list is filenames, not image names, this should be `openjdk`.